process/elasticproc/accounts/accountsProcessor.go

```diff
@@ -101,7 +101,7 @@ func notZeroBalance(balance string) bool {
 }
 
 // PrepareRegularAccountsMap will prepare a map of regular accounts
-func (ap *accountsProcessor) PrepareRegularAccountsMap(timestamp uint64, accounts []*data.Account, shardID uint32) map[string]*data.AccountInfo {
+func (ap *accountsProcessor) PrepareRegularAccountsMap(timestamp uint64, accounts []*data.Account, shardID uint32, timestampMs uint64) map[string]*data.AccountInfo {
```

Comment:
i'm thinking if it might work to pass only `timestampMs` and get old `timestamp` as `timestampMs/1000` when needed, since we care about ms granularity now and we can get seconds granularity from milliseconds

Reply:
in order to avoid passing kind of duplicated variables

Reply:
refactored